lib/fourchette/pgbackups.rb

```diff
@@ -7,22 +10,33 @@ def initialize
   end
 
   def copy(from, to)
-    ensure_pgbackups_is_present(from)
-    ensure_pgbackups_is_present(to)
-
     from_url, from_name = pg_details_for(from)
     to_url, to_name = pg_details_for(to)
 
-    @client =  Heroku::Client::Pgbackups.new pgbackup_url(from) + '/api'
-    @client.create_transfer(from_url, from_name, to_url, to_name)
+    raw_attachment = @heroku.legacy_client.get_attachments(to).body[0]
```

Comment:
it's actually doing a pg:copy - so it doesn't rely on a backup at all. Perhaps this file's name should be changed to reflect that?